README.md

```diff
@@ -51,11 +51,11 @@ default = ["mopro-ffi/circom"]
 [dependencies]
 mopro-ffi = { git = "https://github.com/zkmopro/mopro.git" }
 uniffi = { version = "0.28", features = ["cli"] }
-rust-witness = { git = "https://github.com/vimwitch/rust-witness.git" }
+rust-witness = "=0.1.0"
```

Comment:
Why is this pinned to `=0.1.0`? I think it could be good to do just `0.1.0` so we can push fixes to rust-witness without needing to update mopro